specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2521,49 +2778,49 @@
             "type": "string"
           },
           {
-            "name": "agreementName",
-            "description": "The integration account agreement name.",
+            "name": "batchConfigurationName",
+            "description": "The batch configuration name.",
             "in": "path",
             "required": true,
             "type": "string"
           },
           {
-            "$ref": "#/parameters/api-version"
-          },
-          {
-            "name": "agreement",
-            "description": "The integration account agreement.",
+            "name": "assemblyArtifact",
+            "description": "The assembly artifact.",
```

Comment:
Why is the BatchConfiguration parameter being named and described as an AssemblyArtifact - is this intentional?

Reply:
That was how it was written in the examples I was given, but it doesn't really make sense. I've changed it to be more in line with the function of the call